### DISCUSSION TITLE
Docker in Synology DSM7

### DISCUSSION THREAD

**Discussion by chuckkyle** [Taxel/PlexTraktSync#626] [General]
Has anyone had success getting Plex Trakt Sync to work within docker using the Synology DSM7?  I can get the container running, but validating the API for Plex and Trakt, I am at a stand still.

Thanks

**Comment by DarkenDrekon** [comment]
I just learned about this tool, I would also be interested in getting this working on Synology DSM7.

**Comment by SilverSurferJM** [comment]
Hi both, I realise your questions are a few months old now, but I have this running, on a 6 hourly basis on a DSM7 Synology.

To validate the API you have two choices, either enable SSH and connect via that to run the commands given in the guide here. Alternatively, you can use the terminal window within the Docker GUI on the synology.
This will give you a plextraktsync container which you can manually start at will. It automatically closes after running. Expanding on that, use Control Panel/Task Scheduler to get it running daily on a schedule of your choice.

**Reply by jimmy3vil** [reply]
Hi! I've had this installed for a few weeks, but for the life of me I can't figure out how schedule it to run in Task Scheduler. Would you mind talking through your process?

**Comment by kimpenhaus** [comment]
@jimmy3vil 

I've just setup a simple schduled task which is executing following command:

```
sudo docker start -a <name-of-the-docker-container> && curl -fsS --retry 3 https://my.health.server/ping/some-guid-here > /dev/null
```

all it does is to run the docker container and after that ping the health server and tell it the process is healthy... more simple would be to just run the container

```
sudo docker start -a <name-of-the-docker-container>
```

<img width="535" alt="image" src="https://user-images.githubusercontent.com/575109/183090457-3366ef7d-3301-4a27-b34d-d7f0757943b8.png">

hope that helps

**Comment by smartpippin** [comment]
thanks a lot
